pkg/tsdb/prometheus/models/query.go

```diff
@@ -52,11 +52,12 @@ const (
 var safeResolution = 11000
 
 type QueryModel struct {
-	dataquery.PrometheusDataQuery
+	kinds.PrometheusDataQuery
+	RefId string `json:"refId,omitempty"`
+
 	// The following properties may be part of the request payload, however they are not saved in panel JSON
 	// Timezone offset to align start & end time on backend
 	UtcOffsetSec   int64  `json:"utcOffsetSec,omitempty"`
-	LegendFormat   string `json:"legendFormat,omitempty"`
 	Interval       string `json:"interval,omitempty"`
 	IntervalMs     int64  `json:"intervalMs,omitempty"`
 	IntervalFactor int64  `json:"intervalFactor,omitempty"`
```

Comment:
^^ ???
this exists in the cue model -- but with a float value 🤷🏻 
https://github.com/grafana/grafana/pull/83808/files#diff-fc88a94ee0629cc8c2b03141bf301f3719a22b57c8332fdcd9c24af15aa4d411R44